packages/nextjs/src/config/loaders/sdkMultiplexerLoader.ts

```diff
@@ -13,7 +13,7 @@ type LoaderOptions = {
  * or edge-runtime code.
  */
 export default function sdkMultiplexerLoader(this: LoaderThis<LoaderOptions>, userCode: string): string {
-  if (!userCode.includes('__SENTRY_SDK_MULTIPLEXER__')) {
+  if (!userCode.includes('_SENTRY_SDK_MULTIPLEXER')) {
```

Comment:
m: let's please leave a comment explaining why this is a `exported const` in the doc string